### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by oleg-jukovec**
## Overview

The release adds support for the latest version of the [queue package](https://github.com/tarantool/queue/) with master-replica switching.

## Breaking changes

There are no breaking changes in the release.

## New features

* Support the queue 1.2.1 (#177).
* ConnectionHandler interface for handling changes of connections in ConnectionPool (#178).
* Execute, ExecuteTyped and ExecuteAsync methods to ConnectionPool (#176).
* ConnectorAdapter type to use ConnectionPool as Connector interface (#176).
* An example how to use queue and connection_pool subpackages together (#176).

## Bugfixes

* Mode type description in the connection_pool subpackage (#208).
* Missed Role type constants in the connection_pool subpackage (#208).
* ConnectionPool does not close UnknownRole connections (#208).
* Segmentation faults in ConnectionPool requests after disconnect (#208).
* Addresses in ConnectionPool may be changed from an external code (#208).
* ConnectionPool recreates connections too often (#208).
* A connection is still opened after ConnectionPool.Close() (#208).
* Future.GetTyped() after Future.Get() does not decode response  correctly (#213).
* Decimal package use a test function GetNumberLength instead of a package-level function getNumberLength (#219).
* Datetime location after encode + decode is unequal (#217).
* Wrong interval arithmetic with timezones (#221).
* Invalid MsgPack if STREAM_ID > 127 (#224).
* queue.Take() returns an invalid task (#222).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,17 +10,25 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
...
 ### Added
 
-- Support queue 1.2.0 (#177)
+### Changed
+
+### Fixed
+
+## [1.9.0] - 2022-10-05
+
+The release adds support for the latest version of the
```
Comment (DifferentialOrange) [inline CHANGELOG.md:19]: Do we write short summary messages in CHANGELOG now?

**Review by DifferentialOrange — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (oleg-jukovec) [reply CHANGELOG.md:19]: Before the pull request we have such messages for 1.5.0 and 1.6.0. Summary messages were missed for 1.7.0 and 1.8.0.

We can unify it here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
```
Comment (DifferentialOrange) [inline CHANGELOG.md:46]: I think `changelog: add missing summary messages` is more correct

Comment (oleg-jukovec) [reply CHANGELOG.md:49]: Thank you, fixed!

**Review by DifferentialOrange — APPROVED** [review]

**Review by vr009 — APPROVED** [review]

**Pull request merged**